qiskit/algorithms/optimizers/adam_amsgrad.py

```diff
@@ -193,14 +193,9 @@ def load_params(self, load_dir: str) -> None:
         t = t[1:-1]
         self._t = np.fromstring(t, dtype=int, sep=" ")
 
-    @deprecate_arguments(
-        {
-            "objective_function": "fun",
-            "initial_point": "x0",
-            "gradient_function": "jac",
-        },
-        since="0.19.0",
-    )
+    @deprecate_arg("objective_function", new_alias="fun", since="0.19.0")
```

Comment:
Arguably its long past time where these should have been removed by now!!

Reply:
Hehe :) 

Non-ironic comment: it is really nice now being able to grep for `since="0.19.0"` etc.